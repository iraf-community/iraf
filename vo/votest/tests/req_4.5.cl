#
#  Select DAL results for download.

# Set the test description string.
votest.descr = "Select DAL results for download"

print ("------------------------------------------------------------------")
print ("Req  4.5:  Users shall be able to select which data returned by a")
print ("	   DAL service query is to be downloaded to local disk.")
print ("------------------------------------------------------------------")


fcache init

# Execute the test commands.

string  ofn1, ofn2

ofn1 = mktemp ("tmp$req45")
ofn2 = mktemp ("tmp$req45") // ".fits"

# Get an SIA service result.
getimg ("dss", "m83", size=0.1, output=ofn1, format="raw")

# Select only the FITS data.  Note the use of the column name here
# implies some knowledge of the table schema.
tselect (ofn1, ofn2, expr="Format ?= 'fits'")

# Print the URLs we would like to download.
tdump (ofn2, columns="URL", cdfile="",  pfile="",  datafile="STDOUT")

# Clean up.
delete (ofn1, verify-, >& "dev$null")
delete (ofn2, verify-, >& "dev$null")


