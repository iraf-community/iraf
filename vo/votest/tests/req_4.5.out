------------------------------------------------------------------
Req  4.5:  Users shall be able to select which data returned by a
	   DAL service query is to be downloaded to local disk.
------------------------------------------------------------------
http://skyview.gsfc.nasa.gov/cgi-bin/images?position=204.253829%2C-29.865761&survey=dss2b&pixels=300%2C300&sampler=Clip&size=0.10000000000000002%2C0.10000000000000002&projection=Tan&coordinates=J2000.0&return=FITS                           
http://skyview.gsfc.nasa.gov/cgi-bin/images?position=204.253829%2C-29.865761&survey=dss2r&pixels=300%2C300&sampler=Clip&size=0.10000000000000002%2C0.10000000000000002&projection=Tan&coordinates=J2000.0&return=FITS                           
http://skyview.gsfc.nasa.gov/cgi-bin/images?position=204.253829%2C-29.865761&survey=dss2ir&pixels=300%2C300&sampler=Clip&size=0.10000000000000002%2C0.10000000000000002&projection=Tan&coordinates=J2000.0&return=FITS                          
